lib/fluent/plugin/in_tail.rb

```diff
@@ -593,13 +592,11 @@ def on_notify
           begin
             if @inode != inode || fsize < @fsize
               # rotated or truncated
+              io = FileWrapper.open(@path) rescue nil
```

Comment:
Hi @sonots ,

> Could you tell me when this io object will be closed?

The file descriptor is consumed here:

https://github.com/fluent/fluentd/blob/master/lib/fluent/plugin/in_tail.rb#L384

The first time through ... an io_handler is created here:

https://github.com/fluent/fluentd/blob/master/lib/fluent/plugin/in_tail.rb#L416

The file descriptor remains open until the next file rotation, removal, or fluentd shutdown.  The logic to deal with some of these situations begins here:

https://github.com/fluent/fluentd/blob/master/lib/fluent/plugin/in_tail.rb#L420

Other cases are handled here:

https://github.com/fluent/fluentd/blob/master/lib/fluent/plugin/in_tail.rb#L188

For example, shutdown calls stop_watchers().

> You should ignore only Errno::ENOENT as repeatedly wrote

Sorry, I thought @repeatedly was giving me a choice, and I picked the first option.  I'll update the pull request using the second option.  I'll also rebase against master.

David